code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,297 @@
+/*TODO:
+	PTSD
+*/
+/datum/brain_trauma/psychological
+	var/cure_points = 1000
+	var/current_points = 0
+	var/fearless_mod = 1
+	var/lithiated_mod = 1
+	var/relaxed_mod = 1
+	var/control_mod = 1
+
+/datum/brain_trauma/psychological/on_life()
+	if(HAS_TRAIT(owner, TRAIT_FEARLESS))
+		current_points += fearless_mod
+	if(HAS_TRAIT(owner, TRAIT_LITHIATED))
+		current_points += lithiated_mod
+	if(HAS_TRAIT(owner, TRAIT_RELAXED))
+		current_points += relaxed_mod
+	if(HAS_TRAIT(owner, TRAIT_UNDER_CONTROL))
+		current_points += control_mod
```

Comment:
so, is there no cap to how high you can bing a trauma's HP value if you use the wrong method(s) to treat it?

maybe you should add one of 3*initial(cure_points) or something, just to keep stuff from getting too out of hand

Reply:
well the whole point of this is that it will get harder to cure something the longer someone is incompetent